docs/napari-plugin-guide.md

```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
 ```
 napari -w recOrder-napari
 ```
+## Connect to MicroManager
+Click “Connect to MM”. If the connection succeeds, proceed to calibration. If not, revisit the [microscope installation guide](./microscope-installation-guide.md).
+
+![](./images/connect_to_mm.png)
 
 ## Calibration tab
-The first step in the QLIPP process is to calibrate the liquid crystals. This process involves generating the polarization states and acquiring a background **on an empty FOV**, so begin by placing your sample on the stage and focusing on the surface of the coverslip or well. 
 
-The light path must also be in **Kohler Illumination** to ensure uniform illumination of the sample. [Please follow these steps to setup Kohler illumination.](https://www.microscopyu.com/tutorials/kohler)
+The first step in the acquisition process is to calibrate the liquid crystals and measure a background. In the `recOrder` plugin you will see the following options for controlling the calibration:
```

Comment:
Should we note that this is only necessary for polarization imaging?

Reply:
Good call, fixed.